### ISSUE TITLE
Git config conditional for users or make Git ask for correct user when initializing/cloning a repo.

### ISSUE THREAD

**Issue by dubgeiser** [dubgeiser/tilde#28]
Too often found myself using the wrong e-mail address when starting / cloning a new repo.
Should be possible with a pre- hook to ask for a user e-mailaddress with the default prefilled.

**Comment by dubgeiser** [comment]
The problem still exists, but another possibility might be to have different Git config files for different jobs/roles and use Git configs `[includeif]` that checks for the folder the repo resides in.... ex:
```
[includeif "gitdir:Projects/"]
    path = .gitconfig-work
[includeif "gitdir:OtherProjects/"]
    path = .gitconfig-other
```

**Comment by dubgeiser** [comment]
This is fixed in [0088e75]
